### ISSUE TITLE
Increased urine succinate concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6009]
**Preferred term label:** Increased urine succinate concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Succinate concentration in the urine is increased above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033092
